docs/reference/browser-support.md

```diff
@@ -18,19 +18,19 @@ Stencil builds Web Components that run natively or near-natively in all widely u
   <div class="bs-chart__group">
     <div class="bs-chart__cards">
       <div class="bs-chart__card">
-        <app-icon name="checkmark"></app-icon>
+        <ion-icon name="checkmark"></ion-icon>
```

Comment:
I didn't know we were pulling in Ionic Framework already 🤔

Reply:
yeah we are currently pulling in ionicons here:

https://github.com/ionic-team/stencil-site/blob/40fd73cba2c109baa1c45ed8fb4d7bdc9bf7acc6/docusaurus.config.js#L28-L33

Reply:
I thought, however, that since we use emojis for charts like this on e.g. the [support policy](https://stenciljs.com/docs/next/support-policy) page we might want to make that change on this page too - I did that so we can see what we think for the polyfills chart at the bottom of the page

Reply:
<img width="920" alt="Screenshot 2023-01-26 at 1 21 14 PM" src="https://user-images.githubusercontent.com/6207644/214920034-cc9293dc-1e1c-412b-af08-518bc874a2e0.png">

looks like this